src/main/java/no/nav/pto/veilarbportefolje/kafka/KafkaConfigCommon.java

```diff
@@ -396,6 +389,21 @@ public KafkaConfigCommon(CVService cvService,
                 .withToggle(() -> unleashService.isEnabled(KAFKA_SISTE_14A_STOP) || kafkaAivenUnleash.get())
                 .build();
 
+        consumerClientAivenTiltaksaktivitetendret = KafkaConsumerClientBuilder.builder()
+                .withProperties(aivenDefaultConsumerProperties(CLIENT_ID_CONFIG))
+                .withTopicConfig(new KafkaConsumerClientBuilder.TopicConfig<String, TiltakDTO>()
+                        .withLogging()
+                        .withMetrics(prometheusMeterRegistry)
+                        .withStoreOnFailure(consumerRepository)
+                        .withConsumerConfig(
+                                Topic.TILTAK_TOPIC.topicName,
+                                Deserializers.stringDeserializer(),
+                                Deserializers.jsonDeserializer(TiltakDTO.class),
+                                tiltakService::behandleKafkaRecord
+                        ))
+                .withToggle(() -> unleashService.isEnabled(KAFKA_TILTAKSAKTIVITET_ENDRET_STOP) || kafkaAivenUnleash.get())
+                .build();
+
         consumerRecordProcessor = KafkaConsumerRecordProcessorBuilder
                 .builder()
                 .withLockProvider(new JdbcTemplateLockProvider(jdbcTemplate))
```

Comment:
`consumerRecordProcessor` should include topic config from `consumerClientAivenTiltaksaktivitetendret` (similar to `siste14aTopicConfig`)